pkg/persistence/object_store.go

```diff
@@ -43,9 +43,10 @@ type BackupStore interface {
 
 	ListBackups() ([]string, error)
 
-	PutBackup(name string, metadata, contents, log, volumeSnapshots io.Reader) error
+	PutBackup(name string, metadata, contents, log, podVolumeBackups, volumeSnapshots io.Reader) error
```

Comment:
Wondering if we should define a struct to encapsulate some/all of the arguments to this function, given that we're up to 6 - WDYT?

Reply:
Agree. More then 3 and it's something to think about, in this case it totally makes sense.

Reply:
:+1:. The CSI proposal adds yet another argument for VolumeSnapshots, so I'm for a struct.